release-team/role-handbooks/communications/README.md

```diff
@@ -81,26 +81,51 @@ The release lead will drive the content for the release theme and logo.
 
 Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and are authored by enhancement developers and others close to the features. We do, however, need to encourage owners of important enhancements to opt in to writing feature blogs. Examples of enhancements that warrant a feature blog might include: breaking changes, features and changes important to our users, features that have been in progress for a long time and are graduating, and features that are considered mandatory by the Release Lead. It helps to work closely with the Release Lead and use the respective SIG Slack channels to remind the SIGs about opting in to feature blogs and provide any necessary context to blog authors. 
 
-**The first feature blog** typically goes out on release day alongside or shortly after the release blog, and then are published one-at-a-time, typically at a rate of two to three posts weekly. **The Comms team establishes the publication schedule.**
-> Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.
+**Reach out in KEP issues to ask for feature blog opt-in.** Ask every KEP owner if they want to contribute a blog by reaching out in the KEP issue. Example messaging is below.
+
+```
+👋 from the v1.31 Communications Team!
+We'd love for you to consider writing a feature blog about your enhancement! Some reasons why you might want to write a blog for this feature include (but are not limited to) if this introduces breaking changes, is important to our users, or has been in progress for a long time and is graduating.  
+
+To opt-in, let us know and open a Feature Blog placeholder PR against the [website repository](https://github.com/kubernetes/website) by **[BLOG PLACEHOLDER PR DEADLINE]**. For more information about writing a blog see the [blog contribution guidelines](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#technical-considerations-for-submitting-a-blog-post).
+
+Note: In your placeholder PR, use `XX` characters for the blog `date` in the front matter and file name. We will work with you on updating the PR with the publication date once we have a final number of feature blogs for this release.```
+
+Note that not every KEP needs a blog. Work with the KEP owners, Release Lead, and SIG Docs Blog team (though #sig-dcos-blog slack channel) to make sure all KEPs that opt-in really need a blog written. Alternatives for writing a full blog are including the KEP as a Major Theme, a smaller mention in release announcement
+
+**As feature blogs are opted in and placeholder PRs are created**, assign the blogs to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
```

Comment:
I would add a note to ensure smooth communication with #sig-docs-blogs on Slack, along these lines:

In case the blog you are trackings needs a technical review - we recommend each blog has passed at least a technical review stage - you should contact the specific SIG about it, possibly adding a tag to the author as well.

In case the blog tracked is instead in need of a content, format and style review you should write a message on #sig-docs-blogs asking if someone in the team has bandwidth to help out with such review, possibly tagging the current PR wrangler (see [PR wrangler schedule](https://github.com/kubernetes/website/wiki/PR-Wranglers)).

Note: Comms shadows are welcome to add an informal LGTM message if they feel confident enough to review the blog and if they didn't find any major blockers or nits to fix. The last approval of the PR is still a responsibility of blog approvers but they don't mind if you help out!

Reply:
Resolved!